Orange/widgets/data/owrank.py

```diff
@@ -573,71 +458,31 @@ def autoSelection(self):
         else:
             selection = QItemSelection()
             if len(self.selected_rows):
-                selection = QItemSelection()
-                for row in self.selected_rows:
+                for row in model.mapFromSource(self.selected_rows):
                     selection.append(QItemSelectionRange(
                         model.index(row, 0), model.index(row, columnCount - 1)))
 
         selModel.select(selection, QItemSelectionModel.ClearAndSelect)
 
+        self.setFocus(Qt.OtherFocusReason)  # Prevent graying out scores?
```

Comment:
Similar was there before: https://github.com/kernc/orange3/blob/177ce2a524d592849ee32463455dc2de6d7b2e6e/Orange/widgets/data/owrank.py#L527-L528
I don't quite get what it was supposed to fix. @thocevar ?